frontend/config/agents.ts

```diff
@@ -114,43 +107,53 @@ export const AGENT_CONFIG: {
     displayName: 'Agents.fun agent',
     description:
       'Autonomously post to Twitter, create and trade memecoins, and interact with other agents.',
+    eoaFunding: {
+      [EvmChainId.Base]: {
+        chainConfig: CHAIN_CONFIG[EvmChainId.Base],
+      },
+    },
     isAgentEnabled: true,
   },
   [AgentType.Modius]: {
     name: 'Modius agent',
     evmHomeChainId: EvmChainId.Mode,
     middlewareHomeChainId: MiddlewareChain.MODE,
     requiresAgentSafesOn: [EvmChainId.Mode],
-    agentSafeFundingRequirements: {
-      [EvmChainId.Mode]: 0.0005,
-    },
     additionalRequirements: {
       [EvmChainId.Mode]: {
         [TokenSymbol.USDC]: Number(
           formatUnits(
-            modiusFundRequirements?.[
+            modiusFundRequirements[
               MODE_TOKEN_CONFIG[TokenSymbol.USDC].address as string
-            ]?.safe ?? 0,
+            ].safe,
             MODE_TOKEN_CONFIG[TokenSymbol.USDC].decimals,
           ),
         ),
       },
     },
     operatingThresholds: {
       [WalletOwnerType.Master]: {
+        [WalletType.Safe]: {
+          [TokenSymbol.ETH]: Number(
+            formatEther(
+              `${modiusFundRequirements[ethers.constants.AddressZero].safe}`,
```

Comment:
can you please replace it with `${modiusFundRequirements[ethers.constants.AddressZero].agent}` temporarily and with a todo like [here](https://github.com/valory-xyz/olas-operate-app/blob/release/modius/frontend/constants/serviceTemplates.ts#L294). otherwise we'll show the low funds alert too early. See this for context: https://valory-workspace.slack.com/archives/C07CKA5TN5V/p1736161368676119